src/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:centos8.3.2011
+
+LABEL org.opencontainers.image.title="huawei-csi-plugin"
+LABEL org.opencontainers.image.description="A docker image containing the Huawei CSI plugin for k8s"
+LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
+LABEL org.opencontainers.image.vendor="Adfinis"
+LABEL org.opencontainers.image.url="https://github.com/adfinis-sygroup/huawei-csi-plugin"
+LABEL org.opencontainers.image.licenses="AGPL-3.0-or-later"
```

Comment:
Some of these overlap with the labels in the GitHub Action.

Reply:
This is a copy paste from back8sup